docs/DisplayUpdateSequence.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+autonumber
+
+Application -> DataManager : BuildDisplayData()
+
+DataManager -> LunarRetriever : retrieve
+DataManager <- LunarRetriever : LunarData
+
+DataManager -> TideRetriever : retrieve
+DataManager <- TideRetriever : TideData
+
+DataManager -> WeatherRetriever : retrieve
+DataManager <- WeatherRetriever : WeatherData
+
+Application <- DataManager : DisplayData
+
+Application -> DisplayManager : Render(DisplayData)
+    loop for each DailyPrediction
+
+
+        group Displaying Lunar Data
+            DisplayManager -> DailyPrediction : lunarData.visible
+            note right: Need to confirm this with Christopher
```

Comment:
I'm also using the segment member of the LunarData class to determine
whether or not the moon is waxing or waning.
(I'm assuming this is the file that generates the svgs)
```suggestion
            DisplayManager -> DailyPrediction : lunarData.segment
```
other than that, this looks good to me.